docs/discussions/process.md

```diff
@@ -0,0 +1,19 @@
+# Process #
+
+This doc describes some of our team's processes, ceremonies (a fancy term for meetings), and philosophy about working in an Agile way.
+
+## Predictability ##
+
+Our stakeholders want to see momentum and don't really care about how many story points we get done or what our velocity is, so we would like to find a way to reliably deliver something they care about (mainly components) every week.
+
+## Planning ##
+
+Planning is an agile ceremony where we decide what component we are going to publish in a given week and discuss the logistics necessary to make that happen. Addiitonally, we decide what long-running work is going to be happening in parallel and who is going to work on that.
```

Comment:
> what component we are going to publish in a given week

I wonder if we want to be more general here, to not put us on the hook for a component per week.